### PR TITLE
Reworking of Committee and Task Force Policy pages

### PR DESCRIPTION
**Pull request by anenadic**
Reworking of Committee and Task Force Policy pages to address #700. This work should at least include the following:

- [x] Update ”The Carpentries Committee Policy” page under Governance section:
	- clearly explain who organises and approves them, and what its roles and responsibilities are
	- explain better what should be included in the "short description of what the committee does" (as per [Toby’s comment](https://github.com/carpentries/docs.carpentries.org/issues/700#issuecomment-823380867))
	- explain in more detail how a Committee is dissolved (as per [Toby’s comment](https://github.com/carpentries/docs.carpentries.org/issues/700#issuecomment-823380867))
	- explain the difference between a Committee and a Task Force (as per [Toby’s comment](https://github.com/carpentries/docs.carpentries.org/issues/700#issuecomment-823380867))
	- explain that the EC does not have to approve all committees now 
	- link to “Committees and Task Forces” section in Bylaws (as per [Toby’s comment](https://github.com/carpentries/docs.carpentries.org/issues/700#issuecomment-823380867)).
- [x] Update "The Carpentries Task Force Policy” page under Governance section:
	- clearly explain who organises and approves them, and what its roles and responsibilities
	- explain the difference between a Committee and a Task Force (as per [Toby’s comment](https://github.com/carpentries/docs.carpentries.org/issues/700#issuecomment-823380867))
	- link to “Committees and Task Forces” section in Bylaws.
- [x] Use consistent capitalisation of words for Committees/Task Force (as per [Toby’s comment](https://github.com/carpentries/docs.carpentries.org/issues/700#issuecomment-823380867))
- [x] Address [Erin’s comments](https://github.com/carpentries/docs.carpentries.org/issues/700#issuecomment-823342236) whether maintainers are a committee and relaxing requirements for the minimum number of meetings per year and hosting community discussions for Curriculum Advisory Committees.
- [x] Text in section “Committees and Task Forces” in Bylaws also needs improving/harmonising with the "The Carpentries Committee Policy” and "The Carpentries Task Force Policy” pages - but this comes under Bylaws updates - make a note in the appropriate place that this update is due in 2022.
- [x] Update the “update log”

Note that some changes would be needed in the ByLaws section too:
- Committees and Task Forces are formed either by Core Team or EC
- CarpentryCon TF should be moved out of ByLaws

### CODE REVIEW

**Comment by anenadic** [comment]
There are a couple of issues with the pages and policies that need clarifying:
- the use of should/shall
- enforcing a minimum number of meeting per year
- enforcing a minimum number of reports to present to EC
- enforcing a minimum number of updates to public-facing pages
- not clear in which cases EC does not need to approve a Committee/Task Force and is it a Core Team that approves in such cases

**Comment by anenadic** [comment]
> I started a review but soon realised I'd better start a discussion first. I though the idea was that Committees and Task Forces no longer require EC approval, and that they can be established and terminated by the Core Team. EC may still initiate a C or TF, which obviously would not require CT approval and can only be terminated by the EC.
> 
> I do think what is written under "Actions Requiring Executive Council Approval" for each should remain there.
> 
> Finally, perhaps C and TF formation should be at the top of the texts for each of them, before operations.

Thank you for having a look at this @Lex - and yes, I'd appreciate some discussion around this to clarify a few things. I can do some fixes now based on your initial feedback and see where I get to.

**Comment by anenadic** [comment]
My interpretation of things (after talking to @sstevens2 ), proposed changes and discussion points:
* The main change is that committees and task forces can be proposed **and approved** by the Core Team (previously only EC could approve them). 
* The Bylaws currently say: “Both Committees and Task Forces are overseen by Core Team, although the Executive Council may request quarterly reports on each group’s activities.” 
    * This should be changed so that committees and task forces have to report to the EC (so language needs to be stronger, and this was the case up to now as well in terms of reporting). But in the case that the CT directly runs these groups, the report may go to the CT first then to EC - this however does not need to be written in Bylaws.
*  Frequency of the reports is a bit too strong in the committee and task force policies and is set to “at least 4 per year”. This may be too much as some of these groups may only meet once or twice a year. The proposal is to change this to " twice or at least a year and agreed at the time of the committee/task force formation but with a possibility to update the frequency later on in agreement with the EC".
* Which body terminates committees and task forces?
   * EC approves this formally but the request/proposal should come from the group itself or the CT? Should we say something about naturally ceasing to exist and informing the EC?    
* Can committees be formed/proposed by the community too? Currently, by the Bylaws, only task forces can be proposed by the community and committees only by the EC/CT. A general feeling was no (but cannot remember now why)

**Comment by tobyhodges** [comment]
> We do not have a clear contact info for the community who want to start a task force (or a committee) - perhaps we can have the [team@carpentries.org](mailto:team@carpentries.org) added as contact point to the page: https://carpentries.org/committees/ to start the initial conversations.

Perhaps community members could open an issue on https://github.com/carpentries/task-forces to propose the formation of a new task force?

File: topic_folders/governance/bylaws.md [hunk 1]
```diff
@@ -161,16 +161,29 @@ If an officer steps down, is removed from the Executive Council, or is otherwise
...
 
 ### Task Forces
-Task Forces are organised by the Executive Council to carry out specific tasks on a short-term basis (e.g., a year or less). Formation, Operations and terminations of Task Forces are regulated in the [Task Force Policy](https://docs.carpentries.org/topic_folders/governance/task-force-policy.html).
-
-### CarpentryCon Task Force
-CarpentryCon is the key community-building and networking event in The Carpentries' calendar of activities. This event brings together newer and more experienced community members to share knowledge, to network, to develop new skills, and to develop strategies for building strong local communities. Location and frequency of the meeting is determined by the CarpentryCon Task Force.
+Task Forces are organised to bring together a small group of people focused on a
+particular topic for a set period of time (typically short-term, e.g. a year or less) - 
+to explore ideas, propose or make updates in a policy, procedures or guidelines,
+organise an event or information gathering activity for another decision or challenge.
```
Comment (sstevens2) [inline topic_folders/governance/bylaws.md:184]:
```suggestion
or organise an event or information gathering activity for another decision or challenge. 
```

File: topic_folders/governance/committee-policy.md [hunk 2]
```diff
@@ -1,77 +1,102 @@
...
-* Host branded events
-* Create and/or use a logo
-* Launch their own fundraising campaign
-
-Note that the above list is not exhaustive. Committees should consult with the Executive Council if there is any question about whether or not what they are doing requires approval.
-
-Update log:
-
-Approved [2020-07-06](https://github.com/carpentries/executive-council-info/issues/43) by the Executive Council 
+A committee is initiated by outlining a [proposal](#committee-proposal) and abiding to the following the approval procedure:
```
Comment (sstevens2) [inline topic_folders/governance/committee-policy.md:10]:
```suggestion
A committee is initiated by outlining a [proposal](#committee-proposal) and abiding by the following the approval procedure:
```
maybe an American/British English difference? Not sure.  Ignore if so.

File: topic_folders/governance/committee-policy.md [hunk 3]
```diff
@@ -1,77 +1,102 @@
...
+- explain why it should be an on-going/longer-term project 
+- list aims and objectives of the committee
+
+### Committee Operations 
+The following are operational guidelines for committees.
+
+#### Roles and Responsibilities
+Each committee should:
+ 
+- Create [public documentation for the committee](#public-documentation) described below
```
Comment (sstevens2) [inline topic_folders/governance/committee-policy.md:34]:
```suggestion
- Create the [public documentation for the committee](#public-documentation) described below
```

File: topic_folders/governance/committee-policy.md [hunk 4]
```diff
@@ -1,77 +1,102 @@
...
+- Appoint a committee chair, and, optionally, a co-chair
+- Identify term limits for chair/co-chair/standard members
+- Provide a public “Roles and Responsibilities” document for the committee and its members
+- Create internal documentation (which may be made public), including:
+  - Process for recruiting and/or selecting new members
+  - Process for on-boarding new members
+  - Process for off-boarding members
+  - A succession plan for chair/co-chair
+  - Decision making process
+- Appoint a [Core Team](https://carpentries.org/team/) member liaison to serve as a point of contact and (optionally) attend committee meetings, and
```
Comment (sstevens2) [inline topic_folders/governance/committee-policy.md:44]:
```suggestion
- Request a [Core Team](https://carpentries.org/team/) member liaison to serve as a point of contact and (optionally) attend committee meetings, and 
```

File: topic_folders/governance/committee-policy.md [hunk 5]
```diff
@@ -1,77 +1,102 @@
...
+  - Decision making process
+- Appoint a [Core Team](https://carpentries.org/team/) member liaison to serve as a point of contact and (optionally) attend committee meetings, and 
+communicate with the liaison on a regular basis
+- Meetings and communication:
+  - Organise regular meetings (at least twice a year) and post meeting notes in a consistent location (meeting notes should be public unless the work of
+    the committee requires them to remain private)
+  - Post meetings in the community calendar (note that meetings do not necessarily need to be open to public to attend - for example, the Code of Conduct Committee's 
+business meetings are public but the incident meetings are not)
+  - Aim to host at least one community discussion per year to let the community know about the committee's activities
+  - Utilise other [The Carpentries community communication pathways](https://docs.carpentries.org/topic_folders/communications/index.html) to engage with the community as needed
```
Comment (sstevens2) [inline topic_folders/governance/committee-policy.md:52]:
```suggestion
  - Utilise other [Carpentries community communication pathways](https://docs.carpentries.org/topic_folders/communications/index.html) to engage with the community as needed
```

File: topic_folders/governance/committee-policy.md [hunk 6]
```diff
@@ -1,77 +1,102 @@
...
+Executive Council to clarify if things they plan to do require approval.
+
+### Committee Termination
+A committee may naturally cease to function upon fulfilling its role, in which case the Executive Council 
+should be notified. The Core Team may terminate committees that it had approved. 
+Executive Council may terminate any committee at any time.
+
+### Past & Current Committees
+
+The following is a list of some community committees:
```
Comment (sstevens2) [inline topic_folders/governance/committee-policy.md:90]: some or all?

File: topic_folders/governance/task-force-policy.md [hunk 7]
```diff
@@ -1,69 +1,94 @@
...
-Each task force shall:
-* post meeting notes or minutes in the corresponding task force folder in the task forces GitHub repository. Minutes should be public, but can be private if the work of the task force requires it
-* utilise The Carpentries [communication pathways](https://docs.carpentries.org/topic_folders/communications/index.html)
-* produce a Task Force report to be shared with the Executive Council upon completion of the deliverables as outlined in the Task Force Charter
+This policy defines the operational processes and the lines of communication to and from a task force
+to ensure transparency, accountability and effectiveness the task force's activities.
 
-### Public Documentation
+### Task Force Formation                                             
+A task force is initiated by submitting a [task force charter proposal](#task-force-charter-proposal) and abiding to the following approval procedure:
```
Comment (sstevens2) [inline topic_folders/governance/task-force-policy.md:10]:
```suggestion
A task force is initiated by submitting a [task force charter proposal](#task-force-charter-proposal) and abiding by the following approval procedure:
```
still not sure about this one

File: topic_folders/governance/task-force-policy.md [hunk 8]
```diff
@@ -1,69 +1,94 @@
...
+#### Roles and Responsibilities
+Each task force should:
+- Create the official [Task Force Charter](https://github.com/carpentries/task-forces/blob/master/task-force-charter-template.md) after the proposal is approved
+- Appoint a task force chair, and, optionally, a co-chair
+- Provide a public “Roles and Responsibilities” document for the task force and its members
+- Have a public-facing folder in the [Carpentries Task Forces Repository](https://github.com/carpentries/task-forces) to store the Task Force Charter and all relevant public information about the task force
+- Create internal documentation (which may be made public), including:
+  - Process for recruiting and/or selecting new members
+  - Decision making process
+- Appoint a [Core Team](https://carpentries.org/team/) member liaison to serve as a point of contact and (optionally) attend committee meetings, and
```
Comment (sstevens2) [inline topic_folders/governance/task-force-policy.md:46]:
```suggestion
- Request a [Core Team](https://carpentries.org/team/) member liaison to serve as a point of contact and (optionally) attend committee meetings, and
```

File: topic_folders/governance/task-force-policy.md [hunk 9]
```diff
@@ -1,69 +1,94 @@
...
+  - Post meetings in the community calendar (note that meetings do not necessarily need to be open to public to attend - for example, the Code of Conduct Committee's
+    business meetings are public but the incident meetings are not)
+  - Utilise other [Carpentries community communication pathways](https://docs.carpentries.org/topic_folders/communications/index.html) to engage with the community as needed
+- Prepare a written report to the Executive Council upon completion of the task force's activities/deliverables as outlined in its charter
+
+#### Public Documentation
+Each task force should have a public-facing folder in the 
+[Carpentries Task Forces GitHub Repository](https://github.com/carpentries/task-forces) with the following information:
+
+- [Task Force Charter](https://github.com/carpentries/task-forces/blob/master/task-force-charter-template.md)
```
Comment (sstevens2) [inline topic_folders/governance/task-force-policy.md:59]:
```suggestion
- [Task Force Charter](https://github.com/carpentries/task-forces/blob/master/task-force-charter-template.md)
- Clear  indication about the current status of the Task Force (active, project completed)
```

File: topic_folders/governance/task-force-policy.md [hunk 10]
```diff
@@ -1,69 +1,94 @@
...
+The task force chair is responsible for keeping the public documentation up to date.
+
+#### Actions Requiring Executive Council Approval
+A task force will not undertake any of the following actions without obtaining approval of the Executive Council:
+
+- Any modifications to the Code of Conduct
+- Creation and/or release of position statements
+- Host Carpentries-branded events
+- Any modifications of The Carpentries logos (refer to the [Carpentries Logo Usage Policy](https://docs.carpentries.org/topic_folders/communications/resources/logos.html))
+- Launch an own fundraising campaign
```
Comment (sstevens2) [inline topic_folders/governance/task-force-policy.md:75]:
```suggestion
- Launch a fundraising campaign
```

File: topic_folders/governance/task-force-policy.md [hunk 11]
```diff
@@ -1,69 +1,94 @@
...
+
+### Task Force Termination
+A task force will naturally cease to function upon fulfilling its role. 
+The Core Team may terminate task forces that it had approved.
+Executive Council may terminate any task force at any time.
+
+### Past & Current Task Forces
+The following is a list of some community task forces:
+
+- [CarpentryCon Task Force](https://carpentries.org/carp-con-tf/) - a recurring task force that gets periodically
```
Comment (sstevens2) [inline topic_folders/governance/task-force-policy.md:88]:
```suggestion
- [CarpentryCon Task Force](https://carpentries.org/carp-con-tf/) - a recurring task force that gets created periodically 
```

**Review by sstevens2 — APPROVED** [review]
Some small feedback but overall I'm very happy with these changes.

File: topic_folders/governance/committee-policy.md (hunk 6 above)
Comment (anenadic) [reply topic_folders/governance/committee-policy.md:90]: Put 'some' as the page may get outdated and not list them all

**Review by sstevens2 — APPROVED** [review]

**Comment by kariljordan** [comment]
@anenadic pinging you to see if we're ready to merge this PR.

File: topic_folders/governance/committee-policy.md [hunk 12]
```diff
@@ -1,77 +1,95 @@
...
+
+Upon approval, the committee is listed at [The Carpentries website](https://carpentries.org/committees/)
+with a link to its public documentation.
+
+### Committee Operations
+The following are operational guidelines for committees. Each committee should:
+
+- Create the official [committee charter](https://github.com/carpentries/committees/blob/main/committee-charter-template.md), including the [committee's roles and responsibilities](https://github.com/carpentries/committees/blob/main/committee-charter-template.md#roles-and-responsibilities)
+- Appoint the committee chair, and, optionally, a co-chair and identify term limits for chair/co-chair/standard members
+- Appoint a [Core Team](https://carpentries.org/team/) member liaison to serve as a point of contact and (optionally) attend the committee meetings
```
Comment (sstevens2) [inline topic_folders/governance/committee-policy.md:38]:
```suggestion
- Request a [Core Team](https://carpentries.org/team/) member liaison to serve as a point of contact and (optionally) attend the committee meetings
```

File: topic_folders/governance/task-force-policy.md [hunk 13]
```diff
@@ -1,69 +1,86 @@
...
-* Launch its own fundraising campaign
-
-Note that the above list is not exhaustive. Task forces should consult with the Executive Council if there is any question about whether or not what they are doing requires approval.
+### Task Force Operations
+The following are operational guidelines for task forces. Each task force should:
+
+- Create the official [task force charter](https://github.com/carpentries/task-forces/blob/master/task-force-charter-template.md), 
+including the task force's roles and responsibilities
+- Appoint the task force chair, and, optionally, a co-chair
+- Appoint a [Core Team](https://carpentries.org/team/) member liaison to serve as a point of contact and (optionally) attend the task force meetings
```
Comment (sstevens2) [inline topic_folders/governance/task-force-policy.md:40]:
```suggestion
- Request a [Core Team](https://carpentries.org/team/) member liaison to serve as a point of contact and (optionally) attend the task force meetings
```

**Review by sstevens2 — APPROVED** [review]

**Comment by kariljordan** [comment]
@anenadic please see my comment from October 10th in the [EC repo](https://github.com/carpentries-ec/conversations_ec_ed/issues/134) regarding our path forward for handbooks for community roles. We were recommending not opening another repository as it adds more administrative oversight for the Core Team.

**Comment by anenadic** [comment]
@konrad Hi, the PR is ready to be merged but just wanted to align it with the process on Bylaws updates - can we have a quick chat to decide how best to do this merge?

**Comment by maneesha** [comment]
@anenadic  Can this be closed now that you have put in #881 ?

**Comment by anenadic** [comment]
> @anenadic Can this be closed now that you have put in #881 ?

Yes, I believe so - thank you for checking @maneesha. I do not think I have the right access to close the issue myself though.

**Pull request closed without merging**